OAuth2/Client/Impl/FacebookClient.cs

```diff
@@ -71,7 +71,7 @@ protected override Endpoint UserInfoServiceEndpoint
         /// </summary>
         protected override void BeforeGetUserInfo(BeforeAfterRequestArgs args)
         {
-            args.Request.AddParameter("fields", "id,first_name,last_name,email,picture");
+            args.Request.AddParameter("fields", "id,first_name,last_name,email,picture.width(256).height(256)");
```

Comment:
Is there any reason the width and height is specified here. It seems we want three different image sizes. If this is due to an api change can you please attach a link to the info.